cmds/core/brctl/brctl.go

```diff
@@ -0,0 +1,157 @@
+// Copyright 2024 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+
+	"github.com/u-root/u-root/pkg/brctl"
+)
+
+// cli
+const usage = `
+Usage: brctl [commands]
+commands:
+
+INSTANCES:
+brctl addbr <name>		creates a new instance of the ethernet bridge
+brctl delbr <name>		deletes the instance <name> of the ethernet bridge
+brctl show			show current instance(s) of the ethernet bridge
+
+PORTS:
+brctl addif <brname> <ifname>	will make the interface <ifname> a port of the bridge <brname>
+brctl delif <brname> <ifname>	will detach the interface <ifname> from the bridge <brname>
+brctl show <brname>		will show some information on the bridge and its attached ports
+
+AGEING:
+brctl showmacs <brname> 		shows a list of learned MAC addresses for this bridge
+brctl setageingtime <brname> <time>	sets the ethernet (MAC) address ageing time, in seconds [OPT]
+
+SPANNING TREE PROTOCOL (IEEE 802.1d):
+brctl stp <bridge> <state>			controls this bridge instance's participation in the spanning tree protocol.
+brctl setbridgeprio <bridge> <priority>		sets the bridge's priority to <priority>
+brctl setfd <bridge> <time>			sets the bridge's 'bridge forward delay' to <time> seconds
+brctl sethello <bridge> <time>			sets the bridge's 'bridge hello time' to <time> seconds
+brctl setmaxage <bridge> <time>			sets the bridge's 'maximum message age' to <time> seconds.
+brctl setpathcost <bridge> <port> <cost>	sets the port cost of the port <port> to <cost>. This is a dimensionless metric
+brctl setportprio <bridge> <port> <priority>	sets the port <port>'s priority to <priority>
+brctl hairpin <bridge> <port> <state>		enable/disable hairpin mode on the port <port> of the bridge <bridge>
+`
+
+func run(out io.Writer, argv []string) error {
+	var err error
+	command := argv[0]
+	args := argv[1:]
+
+	switch command {
+	case "addbr":
+		if len(args) != 1 {
+			return fmt.Errorf("too few args")
```

Comment:
maybe move that error on top level like:
```go
var errTooFewArgs = errors.New("too few args")
```

And the return it every time, also it may simply testing with errors.Is.